common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/responses/signup/SignUpContinueApiResult.kt

```diff
@@ -61,7 +72,14 @@ sealed interface SignUpContinueApiResult: ApiResult {
         error = error,
         errorDescription = errorDescription,
         correlationId = correlationId
-    ), SignUpContinueApiResult
+    ), SignUpContinueApiResult {
+        override fun toUnsanitizedString(): String = "AttributesRequired(correlationId=$correlationId, " +
+                "error=$error, errorDescription=$errorDescription, " +
+                "requiredAttributes=$requiredAttributes)"
+
+        override fun toString(): String = "AttributesRequired(correlationId=$correlationId, " +
+                "requiredAttributes=$requiredAttributes)"
```

Comment:
What's the rule applied to the attributes? Currently attributes submitted by developers should not exist in both logs, however, invalid attributes contain PII. Required attributes are not. May I ask the reasons behind?